test/e2e/okta/okta.mocks.ts

```diff
@@ -167,6 +167,11 @@ export const mockValidJWT: (override?: Partial<JWTPayload>, mockGetUser?: boolea
     return createTokenForUser(tokenPayload);
 };
 
+export const mockMicroserviceJWT: () => string = () => {
+    const tokenPayload: JWTPayload = generateRandomTokenPayload({ id: 'microservice', role: 'MICROSERVICE' });
```

Comment:
remove the role, add a "createdAt" date, and test again

Reply:
generateRandomTokenPayload no acepta un createdAt, es un partial JWTPayload lo que acepta.

Reply:
generateRandomTokenPayload does not accept createdAt, it is a partial JWTPayload and it does not include createdAt anywhere.